organisations/service.go

```diff
@@ -96,30 +100,41 @@ func (cd service) Write(thing interface{}) error {
 
 	err, stringType := o.Type.String()
 	if err == nil {
-		statement.WriteString("SET o:" + stringType + " ")
+		setTypeStatement := fmt.Sprintf(`MERGE (o:Thing {uuid: {uuid}})  set o : %s `, stringType)
```

Comment:
I guess this is still using string formatting because we can't parameterise the labels in the neo queries?
If so, fine for now, but I might try to avoid this in another PR later.  I this we should try slightly harder to avoid building strings like this for a number of reasons.
This also ties in with another change I'm thinking about, which is that our type hierarchy information is getting scattered and isn't concise or complete.
(So for now, no issue)